src/components/KeyboardSpacer/KeyboardSpacer.js

```diff
@@ -0,0 +1,128 @@
+// Adapted from https://github.com/Andr3wHur5t/react-native-keyboard-spacer/blob/master/KeyboardSpacer.js
```

Comment:
Can we use [patch-package](https://github.com/ds300/patch-package/issues) instead of just copying this library here?

Reply:
Actually, this library hasn't been updated in 5 years so it might be abandoned

Reply:
hello @roryabraham we are adding this library into our code base with this PR https://github.com/Expensify/App/pull/10162

cc @rushatgabhane  @mountiny

Reply:
Okay, thanks for the context.

Reply:
Should we wait for https://github.com/Expensify/App/pull/10162 and then I can remove the changes related to react-native-keyboard-spacer?

Reply:
No need to wait since this PR is higher priority. Whichever one is merged first, the other just may have to resolve conflicts.

Reply:
Ok the version in https://github.com/Expensify/App/pull/10162 seems better than what I did, so if this one is merged first then we can update #10162 to remove the changes made here.